awswrangler/_utils.py

```diff
@@ -334,6 +334,16 @@ def client(
     ...
 
 
+@overload
+def client(
+    service_name: 'Literal["emr-serverless"]',
+    session: Optional[boto3.Session] = None,
+    botocore_config: Optional[Config] = None,
+    verify: Optional[Union[str, bool]] = None,
+) -> "EMRServerlessClient":
```

Comment:
It's not imported :)

Reply:
Ahhhh, that's why mypy wouldn't pick it up. Thanks!